### DISCUSSION TITLE
Keypad Appearance domain and entity, what's the purpose?

### DISCUSSION THREAD

**Discussion by jtommi** [cgiesche/streamdeck-homeassistant#229] [Help and Support] [answered]
So I just got a Streamdeck and of course installed this plugin.
I made the Keypad actions work, but I don't understand what's the point of choosing an entity in the Keypad Appearance.
It doesn't seem to have any impact on the appearance of the button

**Comment by cgiesche** [comment, marked as answer]
Depending on the domain, the button may show different icons for different entity states. If you select a specific entity of a light domain, the state (on / off) of that entity should be displayed of the button.
If you choose a temperature entity, the button shows a thermometer and the current temperature of that entity.

Besides that: Please note, thar the latest release has a bug #227 that prevents selecting an entity in the actions settings. I will release a bug fixed version after the holidays. A possible workaround is mentioned here: #230

**Reply by jtommi** [reply]
Thanks a lot.
After writing my question, I realized it could be used to display sensor states, but somehow I didn't catch that for lights (which I configured) it also shows their state.
